src/components/MyAccount/NewSettings/AddButton.tsx

```diff
@@ -1,15 +1,15 @@
 import { Button } from "@nypl/design-system-react-components"
 
 type AddButtonProps = {
-  inputType: string
+  inputType?: string
   label: string
   onClick: () => void
 }
 
 const AddButton = ({ inputType, label, onClick }: AddButtonProps) => {
   return (
     <Button
-      id={`add-${inputType}-button`}
+      id={inputType ? `add-${inputType}-button` : "add-button"}
```

Comment:
I think it's a good idea to have the id be descriptive. What do you think about reverting this change and adding an input type to the username AddButton?

Reply:
I'm hesitant to add another prop to UsernameForm that would only be used in this one place

Reply:
Maybe i should just pass id directly to AddButton instead of setting like this?

Reply:
Just leave it as is. This is a totally fine pattern.